charts/ccd-message-publisher/values.preview.template.yaml

```
@@ -8,18 +8,17 @@ java:
     DATA_STORE_DB_USERNAME: "{{ .Values.postgresql.auth.username }}"
     DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.auth.password }}"
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
-    CCD_CASE_EVENTS_DESTINATION: ccd-case-events
+    CCD_CASE_EVENTS_DESTINATION: ccd-events
   secrets:
     SERVICE_BUS_CONNECTION_STRING:
-      secretRef: servicebus-secret-topic-{{ .Release.Name }}-servicebus-ccd-case-events
+      secretRef: ccd-sb-preview
       key: connectionString
 
 servicebus:
   enabled: true
-  teamName: CCD
-  resourceGroup: ccd-aks
-  serviceplan: standard
+  resourceGroup: ccd-aso-preview-rg
+  sbNamespace: ccd-sb-preview
```

Can you just update this to
```suggestion
  sbNamespace: ccd
```
please

---

why?
this isn't the namespace we are using